corehq/apps/styleguide/templates/styleguide/bootstrap5/migration_guide.html

```diff
@@ -349,10 +349,101 @@ <h2 id="update-stylesheets" class="pt-4">
+          <div class="card-body">
+            <pre class="m-0">// LESS
+@thing: 20px;
+.@{my-prefix}-kind-of-text: 10px;
+height: calc(~"100vh - 65px");
+
+// SCSS
+$thing: 10px;
+.#{$my-prefix}-kind-of-text: 10px;
+height: calc(100vh - 65px);  // tildes typically weren't necessary</pre>
+          </div>
+        </div>
+      </li>
+      <li>
+        Mixins
+        <div class="card text-bg-light mb-3">
+          <div class="card-body">
+            <pre class="m-0">// LESS
+.my-mixin(@param1, @param2) { ... }
+.my-mixin(170px, 115px);
+
+// SCSS
+@mixin my-mixin { ... }
+@include my-mixin(170px, 115px);</pre>
+          </div>
+        </div>
+      </li>
+      <li>
+        Removing bootstrap 3 browser support mixins, including <code>.transition</code>, <code>.box-shadow</code>, and border radius mixins.
+        <div class="card text-bg-light mb-3">
+          <div class="card-body">
+            <pre class="m-0">// LESS
+.box-shadow(0 0 10px 0 rgba(0, 0, 0, 0.2));
+
+// SCSS
+box-shadow: 0 0 10px 0 rgba(0, 0, 0, 0.2);</pre>
+          </div>
+        </div>
+      </li>
+      <li>
+        Updating responsive styling that depends on screen widths:
+        <div class="card text-bg-light mb-3">
+          <div class="card-body">
+            <pre class="m-0">// LESS
+@media(min-width: @screen-xs) { ... }
+@media(max-width: $screen-sm-max) { ... }
+
+// SCSS
+@include media-breakpoint-up(sm) { ... }
+@include media-breakpoint-down(lg) { ... }</pre>
+          </div>
+        </div>
+        Note that the screen size categories have changed from bootstrap 3 to 5, so see the
+        <a target="_blank" href="https://getbootstrap.com/docs/3.3/css/#responsive-utilities-classes">old docs</a>
+        and the
+        <a target="_blank" href="https://getbootstrap.com/docs/5.0/layout/breakpoints/">new docs</a>, and test
+        your changes. Also note that using the B5 mixins requires imports:
+        <div class="card text-bg-light mb-3">
+          <div class="card-body">
+            <pre class="m-0">@import "mixins";
+@import "functions";</pre>
+      </li>
+      <li>
+        Updating imports. Look at how imports are done in other parts of HQ that use SCSS, such as
+        <a target="_blank" href="https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/hqwebapp/static/hqwebapp/scss/commcarehq.scss">hqwebapp</a> and
+        <a target="_blank" href="https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/hqwebapp/static/cloudcare/scss/formplayer-common-main.scss">cloudcare</a>.
+        <div class="card text-bg-light mb-3">
+          <div class="card-body">
+            <pre class="m-0">// LESS
+@import '../../hqwebapp/less/_hq/includes/variables.less';
+@b3-import-variables: '../../../../../../../../node_modules/bootstrap/less/variables';
+
+// SCSS - test which imports are necessary so that you include the minimum set
+@import "../../hqwebapp/scss/commcarehq/variables";
+@import "variables";
```

Comment:
What is this line doing? It doesn't look like there is an equivalent in the `LESS` example above.

Reply:
Imports aren't necessarily line-for-line equivalent. I can make that explicit, or maybe just remove the code example here.